examples/For arcgishub library/.ipynb_checkpoints/Searching and Filtering content from your site-checkpoint.ipynb

```diff
@@ -0,0 +1,383 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Searching and Filtering content from your site\n",
+    "\n",
+    "In this user guide we explore how the sites.py submodule can be used to search for content across your site. Searched results can be filtered by a search query as well as item type, as we will see in the example that follows.\n",
+    "\n",
+    "We start by connecting to our Hub."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Enter password: ········\n"
+     ]
+    }
+   ],
+   "source": [
+    "from arcgishub.hub import Hub\n",
+    "myhub = Hub(\"https://www.arcgis.com\", \"mmajumdar_dcdev\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In order to search across a site, we first need to fetch the site object. In this example, we fetch the [OpenData site of Washington, D.C.](https://opendata.dc.gov)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "AttributeError",
+     "evalue": "'NoneType' object has no attribute 'typeKeywords'",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[1;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
+      "\u001b[1;32m~\\AppData\\Local\\Temp\\1\\ipykernel_23116\\3436182706.py\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[1;32m----> 1\u001b[1;33m \u001b[0mdc\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mmyhub\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0msites\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mget\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34m'3adff78f286e4ecf936bdefdfeca9890'\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m      2\u001b[0m \u001b[0mdc\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
```

Comment:
why is this error here in the notebook?

it looks like some of the rest of the script didn't work.